test/google/gax/configuration_test.rb

```diff
@@ -88,7 +88,11 @@
     end
 
     it "can inspect" do
-      assert_equal "#<Google::Gax::Configuration {:k1=>{:k2=>{:k3=>{}}}}>", simple_config.inspect
+      expected_inspect = "#<Google::Gax::Configuration: k1=" \
+        "<Google::Gax::Configuration: k2=" \
+        "<Google::Gax::Configuration: k3=" \
+        "<Google::Gax::Configuration: >>>>"
```

Comment:
This seems less useful than the hash-like representation

Reply:
Agreed. I'll change it.

Reply:
Changed.